src/components/VBottomSheet/VBottomSheet.js

```diff
@@ -3,6 +3,8 @@ import '../../stylus/components/_bottom-sheets.styl'
 import VDialog from '../VDialog/VDialog'
 
 export default {
+  functional: true,
```

Comment:
Do you know that u can't import functional components as Async Component (e.g. VDivider => import('path/to/VDivider.vue')). Server Renderer throw an error because he has no `componentOptions` and `Ctor` object.

Reply:
That was fixed in 2.5.16: vuejs/vue#7784